src/OrchardCore.Modules/OrchardCore.Forms/Drivers/FormInputElementPartDisplayDriver.cs

```diff
@@ -23,6 +33,11 @@ public async override Task<IDisplayResult> UpdateAsync(FormInputElementPart part
 
             if (await updater.TryUpdateModelAsync(viewModel, Prefix))
             {
+                if (String.IsNullOrWhiteSpace(viewModel.Name))
+                {
+                    updater.ModelState.AddModelError(Prefix, nameof(viewModel.Name), S["A value is required for Name."]);
```

Comment:
```suggestion
                    updater.ModelState.AddModelError(Prefix, nameof(viewModel.Name), S["The field {0} is required.", nameof(viewModel.Name)]);
```

Reply:
I understand what you mean, for that, we might need to change the data annotation messages too because they rely on params. Something related that there's an issue related to these params in #13520

@rjpowers10 what if we rely on `ErrorMessage` regardless of params